quay-enterprise/prometheus.md

```diff
@@ -0,0 +1,19 @@
+# Prometheus metrics under Quay Enterprise
+
+Quay Enterprise exports a [Prometheus](https://prometheus.io/)-compatible endpoint on each instance to allow for easy monitoring and alerting.
+
+## Exposing the prometheus endpoint
+
+The Prometheus-compatible endpoint on the Quay Enterprise instance can be found at port `9092`. Simply add `-p 9092:9092` to the `docker run` command (or expose the port via the `Pod` configuration in Kubernetes).
+
+## Setting up Prometheus to consume metrics
+
+In order for Prometheus to pull all the metrics cluster-wide, it needs to be aware of the full set of Quay Enterprise instances running. In the typical setup, this is done by placing all the Quay Enterprise instances into a single named DNS entry, which is then given to Prometheus.
```

Comment:
"Prometheus needs a way to access all Quay Enterprise instances running in a cluster." (replace first sentence)